runtime/lua/vscode.lua

```diff
@@ -1,3 +1,13 @@
+-- Preload all vscode-neovim modules
+-- Some plugin managers customize the loader, which can cause issues
```

Comment:
what plugin manager? if they fuck with the loader and break things, that's their fault.

Reply:
From the error message in the issue, lazy.nvim is being used, and lazy.nvim indeed has a custom loader, but I haven't looked into it deeply.

Reply:
lazy.nvim has worked for me and the hundreds of other people using lazy.nvim with vscode neovim

Reply:
Yeah, I'm also using lazy.nvim. But that doesn't mean it won't go wrong.